vbap.h

```diff
@@ -28,24 +28,21 @@
 #define MAX_LS_AMOUNT 55    // maximum amount of loudspeakers, can be increased, but see comments next to MAX_LS_SETS above
 #define MIN_VOL_P_SIDE_LGTH 0.01  
 
-#define VBAP_VERSION "vbap - v1.1 - 14 Aug. 2014 - (c) Ville Pulkki 1999-2006 (Pd port by HCS)"
-#define DFLS_VERSION "define_loudspeakers - v1.0.3.2 - 20 Nov 2010 - (c) Ville Pulkki 1999-2006"
+#define VBAP_VERSION "vbap v1.2 - 17 Jul 2018 - (c) Ville Pulkki 1999-2006 (Pd port by HCS)"
+#define DFLS_VERSION "define_loudspeakers v1.2 - 17 July 2018 - (c) Ville Pulkki 1999-2006"
```

Comment:
same here. I'd go and define a single `VBAP_VERSION=1.2` and use that... (btw, where are the dates coming from?)

Reply:
Oh, that's when I did the actual work last summer. This is post-release cleanup.

Reply:
My reasoning was: "I am a user and I load this object. I expect to see the date when the project was last updated, not the last attribution date."